### PR TITLE
Fixes #734: added a warning about unsaved changes

### LINKED ISSUE #734: [feature request] warn user if leave page without saving
**Is your feature request related to a problem? Please describe.**
In case a user accidentally press previous button in the browser and content is not saved, he loose it all.
**Describe the solution you'd like**
If content (especially how-to) is not saved, show a confirmation alert() pop-up

### PR DESCRIPTION
**Pull request by lauvrenn**
closes #734

### CODE REVIEW

**Comment by BenGamma** [comment]
Nice one with the message ! I'll fix the conflicts myself and will merge. Thanks 🙌

**Comment by lauvrenn** [comment]
@BenGamma I think there's a problem though, as it was based on #757. The Prompt here depends on the upload status of how-tos and it has to be reset somewhere. That's also why the upload status panel in #757 was showing up.

**Comment by lauvrenn** [comment]
ok, I added the reset next to the changes from your 6a0da09fd8ea0ed7912556854f7b5dcbb5273e4c solution, should work now :ok_hand:

File: src/pages/Howto/Content/Common/Howto.form.tsx [hunk 1]
```diff
@@ -120,9 +120,18 @@ export class HowtoForm extends React.Component<IProps, IState> {
...
           <HowToSubmitStatus
             {...this.props}
-            onClose={() => this.setState({ showSubmitModal: false })}
+            onClose={() => {
+              this.setState({ showSubmitModal: false })
+              this.injected.howtoStore.resetUploadStatus()
+            }}
           />
         )}
+        <Prompt
```
Comment (drydenwilliams) [inline src/pages/Howto/Content/Common/Howto.form.tsx]: Cool component, didn't know about this 👍

**Comment by BenGamma** [comment]
Thanks ! Nice and easy.
In an ideal world we would display the Prompt only if the form values have changed, but that's less urgent. The important thing is that the values that the user entered in his how-to doesn't disapear in case of wrong manipulation.
I created a separate issue for that later #766

**Pull request merged**